web/beacon-app/src/components/layout/Topbar/Topbar.tsx

```diff
@@ -1,36 +1,63 @@
 import { memo, ReactNode, useId } from 'react';
 
+import { ProfileCard } from '@/components/common/ProfileCard/ProfileCard';
 import Breadcrumbs from '@/components/ui/Breadcrumbs';
+import { useAuth } from '@/hooks/useAuth';
 import useBreadcrumbs from '@/hooks/useBreadcrumbs';
 
 import ScheduleOfficeHours from '../../ScheduleOfficeHours/ScheduleOfficeHours';
 import MobileNav from '../MobileNav/MobileNav';
 import { Header } from './Topbar.styles';
-
 type TopBarProps = {
   Breadcrumbs?: ReactNode;
+  isOnboarded?: boolean;
+  profileData?: any;
 };
 
-function Topbar({ Breadcrumbs: CustomBreadcrumbs }: TopBarProps) {
+function Topbar({ Breadcrumbs: CustomBreadcrumbs, isOnboarded, profileData }: TopBarProps) {
   const { items, separator } = useBreadcrumbs();
   const id = useId();
+  const { logout } = useAuth();
+
+  const Logout = () => {
+    logout();
+    window.location.href = '/';
+  };
 
   return (
     <>
       <Header className="flex flex-col-reverse items-baseline justify-center gap-2 bg-[#1D65A6] py-2 md:ml-[250px] md:min-h-[60px] md:border-b md:bg-white">
         <div className="flex w-11/12 justify-between">
-          {CustomBreadcrumbs ? (
-            CustomBreadcrumbs
+          {isOnboarded ? (
+            <>
+              {CustomBreadcrumbs ? (
+                CustomBreadcrumbs
+              ) : (
+                <Breadcrumbs separator={separator} className="ml-4 hidden md:block">
```

Comment:
The breadcrumbs have moved more to the left on the dashboard page. Here's a link to a screenshot: https://www.awesomescreenshot.com/image/42930880?key=e0b9432e5f2936d3fd9bc1cb323e7d49